build/crds/grouping/grouping_v1alpha1_nodegroup.yaml

```diff
@@ -0,0 +1,94 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: nodegroups.grouping.kubeedge.io
+spec:
+  group: grouping.kubeedge.io
```

Comment:
The group name needs more discussions, it's more related to apps.

Reply:
🤔 how about `appgroupings`?